mesos/src/cgcloud/mesos/mesos_box.py

```diff
@@ -48,12 +48,14 @@ def mesos_service( name, *flags ):
                             '--registry=in_memory',
                             # would use "--ip mesos-master" here but that option only supports
                             # IP addresses, not DNS names or /etc/hosts entries
-                            '--ip_discovery_command="hostname -i"' ) ],
+                            '--ip_discovery_command="hostname -i"',
+                            '--authenticate_http=true',
```

Comment:
Make this conditional such that it is only added if the Mesos version supports it. Use `_mesos_version` to drive that decision.